package.json

```diff
@@ -31,6 +31,7 @@
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
     "rxjs": "^6.5.4",
+    "soap": "^0.42.0",
```

Comment:
remove soap as we should not be calling CDX services from here use the Auth API validate